internal/gcsx/random_reader_test.go

```diff
@@ -222,6 +222,28 @@ func (t *RandomReaderTest) ExistingReader_WrongOffset() {
 	AssertNe(nil, err)
 }
 
+func (t *RandomReaderTest) ExistingReader_WrongOffsetButWithinRange() {
```

Comment:
I wouldn't call it WrongOffset :)

Reply:
Renamed to `ExistingReader_ReadAtOffsetAfterTheReaderPosition`